<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogModifyInfo</class>
 <widget class="QDialog" name="DialogModifyInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>信息修改</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QDialog {
        background-color: #f5f5f5;
        font-family: &quot;Segoe UI&quot;;
        font-size: 14px;
    }
    QGroupBox {
        border: 2px solid #0078d7;
        border-radius: 8px;
        margin-top: 10px;
        padding-top: 15px;
        font-size: 16px;
        font-weight: bold;
        color: #0078d7;
    }
    QLabel {
        color: #333333;
    }
    QLineEdit {
        border: 1px solid #cccccc;
        border-radius: 4px;
        padding: 5px;
        font-size: 14px;
        min-height: 28px;
    }
    QLineEdit:disabled {
        background-color: #eeeeee;
        color: #666666;
    }
    QLineEdit:focus {
        border: 1px solid #0078d7;
    }
    QPushButton {
        background-color: #0078d7;
        color: white;
        border: none;
        border-radius: 4px;
        padding: 8px 16px;
        min-width: 80px;
        min-height: 34px;
        font-size: 14px;
    }
    QPushButton:hover {
        background-color: #005bb5;
    }
    QPushButton:pressed {
        background-color: #004080;
    }
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>个人信息</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>15</number>
      </property>
      <property name="verticalSpacing">
       <number>10</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="photo">
        <property name="styleSheet">
         <string notr="true">background-color: white; border: 1px dashed #cccccc;</string>
        </property>
        <property name="text">
         <string>[照片预览]</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="noLabel">
        <property name="text">
         <string>工号：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="noLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>姓名：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="nameLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="genderLabel">
        <property name="text">
         <string>性别：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="genderLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="departmentLabel">
        <property name="text">
         <string>部门：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="departmentLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="positionLabel">
        <property name="text">
         <string>职位：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="positionLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="telephoneLabel">
        <property name="text">
         <string>电话：</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="telephoneLineEdit"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="emailLabel">
        <property name="text">
         <string>邮箱：</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="emailLineEdit">
        <property name="placeholderText">
         <string>example@domain.com</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <spacer name="horizontalSpacerLeft">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="BtnOK">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerRight">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
